core/src/main/java/com/linecorp/armeria/client/HttpResponseWrapper.java

```diff
@@ -62,21 +64,25 @@ class HttpResponseWrapper implements StreamWriter<HttpObject> {
     private boolean done;
     private boolean closed;
 
-    HttpResponseWrapper(DecodedHttpResponse delegate, EventLoop eventLoop, ClientRequestContext ctx,
+    HttpResponseWrapper(@Nullable AbstractHttpRequestHandler requestHandler,
+                        DecodedHttpResponse delegate, EventLoop eventLoop, ClientRequestContext ctx,
                         long responseTimeoutMillis, long maxContentLength) {
+        this.requestHandler = requestHandler;
         this.delegate = delegate;
         this.eventLoop = eventLoop;
         this.ctx = ctx;
         this.maxContentLength = maxContentLength;
         this.responseTimeoutMillis = responseTimeoutMillis;
     }
 
-    DecodedHttpResponse delegate() {
-        return delegate;
+    void handle100Continue(ResponseHeaders responseHeaders) {
+        if (requestHandler != null) {
```

Comment:
Question) If a http1 websocket client sends a request with `100-continue`, will it be stuck in a pending state?

I'm curious if we want to just allow websockets as well, or explicitly throw.

Reply:
I prefer throwing an exception. 😉 Thanks!